main.go

```diff
@@ -43,88 +49,106 @@ func initClientConnection() bpb.BirthdayFunctionsClient {
 		log.Fatalln("connection error: ", err)
 		os.Exit(4)
```

Comment:
remove this

Reply:
done

Reply:
You need to add fatal in this case. Because if you  can't connect to the birthday service it is critical. I meant here to delete the os.exit

Reply:
done